### PR TITLE
ci: Fix typecheck on master (no-changelog)

### PR DESCRIPTION
**Pull request by ivov**
https://github.com/n8n-io/n8n/actions/runs/10009425149/job/27668298305

### CODE REVIEW

**Comment by netroy** [comment]
Did it slip through because of the turborepo cache, or because the PR that changed this like wasn't up to date?

**Comment by ivov** [comment]
> Did it slip through because of the turborepo cache, or because the PR that changed this like wasn't up to date?

If it's the latter I'd've expected it to be caught

File: packages/editor-ui/src/composables/useCanvasOperations.ts [hunk 1]
```diff
@@ -1443,7 +1443,7 @@ export function useCanvasOperations({
 
 			if (
 				nodeSaveData.credentials &&
-				settingsStore.isEnterpriseFeatureEnabled(EnterpriseEditionFeature.Sharing)
```
Comment (netroy) [inline packages/editor-ui/src/composables/useCanvasOperations.ts:1446]: I see 29 other instances of `settingsStore.isEnterpriseFeatureEnabled` in the code. can we fix this in the `settingsStore` instead?

**Review by netroy — CHANGES_REQUESTED** [review]

File: packages/editor-ui/src/composables/useCanvasOperations.ts (hunk 1 above)
Comment (ivov) [reply packages/editor-ui/src/composables/useCanvasOperations.ts:1446]: @elsmr Can you please give a hand here as you're most familiar with the context of these changes?

Comment (elsmr) [reply packages/editor-ui/src/composables/useCanvasOperations.ts:1446]: Yep I'll fix this

Comment (elsmr) [reply packages/editor-ui/src/composables/useCanvasOperations.ts:1446]: `isEnterpriseFeatureEnabled` changed from a method to a computed in https://github.com/n8n-io/n8n/pull/10022
My PR (https://github.com/n8n-io/n8n/pull/10112) with changes to `useCanvasOperations` was not up-to-date with this change in master.

Why the `ParameterInputWrapper` didn't fail on Ricardo's branch I'm not sure 😕 , set the settings store state correctly in the test and it works again.

**Review by netroy — APPROVED** [review]
Thanks @elsmr and @ivov for fixing this so quickly on a Friday evening 🙇🏽

**Comment by janober** [comment]
Got released with [`n8n@1.52.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.52.0)

**Pull request merged**